src/pages/developers/services/_meta.json

```
@@ -22,5 +22,9 @@
   "subquery": {
     "title": "SubQuery",
     "description": "Open-source, flexible, multi-chain data indexer"
+  },
+  "goldrush": {
+    "title": "GoldRush API",
```

Let's just keep the project name.

```suggestion
    "title": "GoldRush",
```